js/components/graph/Edge.js

```diff
@@ -20,7 +20,18 @@ export default class Edge extends React.Component {
     if (source === undefined || target === undefined) {
       return
     }
-    if (!source.isSelected() && target.state.status === "missing") {
+
+    if (
+      !source.isSelected() &&
+      target.props.className === "bool" &&
+      target.props.JSON.status === "missing"
+    ) {
+      this.setState({ status: "missing" })
```

Comment:
It's not clear why you changed this if statement, since the code (`this.setState(...)`) in this branch is identical to the code in the following else if branch.